### ISSUE TITLE
Question about usage

### ISSUE THREAD

**Issue by theAkito** [xerpi/vita-udcd-uvc#64]
I'm probably just doing something wrong, so please forgive this issue if this is an easy fix.

I got the plugin in the correct directory, where all the other plugins are working fine, and I loaded it in the config.txt, where there is already another plugin working fine, so that part is correct.

However, after installation and reboot, the PS Vita still gets recognized as a PS Vita on Windows 10 2004. It does not get recognized as an Imaging Device, as seen in the Device Manager.

What am I missing?

P.S.: 
Using a PSV-1000 3.60 with only super basic plugins, like storage manager, pngshot and of course the exploits for 3.60, but nothing else.

**Comment by xerpi** [comment]
It looks like the plugin isn't being loaded correctly, maybe placed it on wrong ux0 or ur0, or the path in the config.txt is wrong,e etc.
Check https://github.com/xerpi/vita-udcd-uvc/issues/12

**Comment by theAkito** [comment]
@xerpi 

I read the whole issue you referenced and checked for corresponding errors. I checked everything 5 times now. Still not working. Also I have another real webcam attached to the computer and it is working. So can't be related to wrong Windows 10 settings or anti-virus, either.

The storage manager plugin is a Kernel plugin, too, which works. So can't be related to my plugin setup, either.

**Comment by xerpi** [comment]
I don't think that special driver conflicts but remove it just in case. I don't really now what could be going one, check if somebody has had a similar issue: https://github.com/xerpi/vita-udcd-uvc/issues
